cogs/poll_command.py

```diff
@@ -64,22 +64,34 @@ async def transform(self, interaction: discord.Interaction, date_time: str) -> d
 
 
 class PollCreate(commands.Cog):
+    POLL_PARAMETERS = {
```

Comment:
Tohle nakonec potřeba není, můžeš toto smazat. Přijdeme na jiný způsob jak to udělat, teď bychom se měli zaměřit jen na vytvoření anket, kdy se anketa smaže a pak pošle embed s hlasy.